apps/dispatch/src/app/components/home/home.component.ts

```diff
@@ -11,6 +11,7 @@ import { DISPATCH_APP_ROUTES } from '../../app.component';
 export class HomeComponent implements OnInit {
   public isMobileHomeDispatch = false;
   public appRoutes = DISPATCH_APP_ROUTES;
+  public logoPath = 'assets/images/runbiosimulations-logo/logo.svg';
```

Comment:
general comment - hard coded asset paths - does the home.component.spec.ts fail if it can't find the file?   This is more of a future looking comment than a request for change.

Reply:
Great question. Currently the relative `dispatch.home.component.spec.ts` uses `compileComponents()` to construct the component from the ancestral templates/styles and validates construction with:

` it('should create', () => {`
`    expect(component).toBeTruthy();`
 ` });`

So to answer your question, the test does not explicitly look for this file but will indeed fail if unable to get/set the value.

Reply:
cool, addressed !!!